### PR TITLE
Added modals for enrollment import, user import and and courses' person management pages

### LINKED ISSUE #1389: Redesign import
When importing data, the import should only be done after an explicit confirmation in a modal (similar to the confirmation when approving evaluations).
The process then would be:
- Upload file for testing
- Resolving errors if needed and re-upload for testing
- Click on "Import previously uploaded file"
- Confirm in a modal

In addition, the colors of the buttons should be changed: The import button should stay `btn-primary` and the test button should always be a `btn-light`.

This should be done for the enrollment import, the user import, and on the Courses' person management pages.

### PR DESCRIPTION
**Pull request by ScrappyDieRaupe**
Fixed #1389.

### CODE REVIEW

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 1]
```diff
@@ -91,7 +91,7 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'From other evaluation' %}</h
                         {% include 'bootstrap_form.html' with form=contributor_copy_form wide=True %}
                     </div>
                     <div class="card-footer text-center">
-                        <button name="operation" value="copy-contributors" type="submit" class="btn btn-sm btn-primary form-submit-btn">{% trans 'Copy contributors' %}</button>
+                        <button name="operation" value="copy-contributors" type="button" onclick="copyContributorsModalShow('copy-contributors', '{{ field }}');" class="btn btn-sm btn-primary form-submit-btn">{% trans 'Copy contributors' %}</button>
```
Comment (He3lixxx) [inline evap/staff/templates/staff_evaluation_person_management.html:94]: What is `{{ field }}` here? As far as I can see, this should be undefined. In the other copy modal above, you pass no second argument.

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 2]
```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
     </div>
 {% endblock %}
 
+{% block modals %}
+{{ block.super }}
+    {% trans 'Import participants' as title %}
+    {% blocktrans asvar question %}Do you really want to upload the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
```
Comment (He3lixxx) [inline evap/staff/templates/staff_evaluation_person_management.html:126]: "upload" is a bit inaccurate here. In the import run, nothing is uploaded anymore.

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 3]
```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
...
+            var input = $("<input>")
+            .attr("type", "hidden")
+            .attr("name", "operation").val(dataId);
+            $('#participant-copy-form').append(input);
+            $('#participant-copy-form').submit();  
+        };
+    </script>
+
+    {% trans 'Import contributors' as title %}
+    {% blocktrans asvar question %}Do you really want to upload the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
```
Comment (He3lixxx) [inline evap/staff/templates/staff_evaluation_person_management.html:154]: "upload" again.

File: evap/staff/templates/staff_user_import.html [hunk 4]
```diff
@@ -20,8 +20,8 @@ <h3>{% trans 'Import users' %}</h3>
             <div class="card-body">
                 <p>
                     {% trans 'Upload Excel file' %}
-                    (<a href="{% url 'staff:download_sample_xls' 'sample_user.xls' %}">{% trans 'Download sample file' %}</a>,
-                    <button type="button" class="btn-link" onclick="copyHeaders(['Title', 'First name', 'Last name', 'Email'])">{% trans 'Copy headers to clipboard' %}</button>).
+                    (<a href="{% url 'staff:download_sample_xls' 'sample_user.xls' %}">{% trans 'download sample file' %}</a>,
+                    <button type="button" class="btn-link" onclick="copyHeaders(['Title', 'First name', 'Last name', 'Email'])">{% trans 'copy headers to clipboard' %}</button>).
```
Comment (janno42) [inline evap/staff/templates/staff_user_import.html:24]: please revert these changes to keep it consistent with other occurrences

File: evap/staff/templates/staff_user_import.html [hunk 5]
```diff
@@ -33,13 +33,31 @@ <h3>{% trans 'Import users' %}</h3>
...
                 {% endif %}
             </div>
         </div>
     </form>
 {% endblock %}
 
+{% block modals %}
+{{ block.super }}
+    {% trans 'Import Users' as title %}
+    {% blocktrans asvar question %}Do you really want to import <strong id="data-label"></strong> from the Excel file? {% endblocktrans %}
```
Comment (janno42) [inline evap/staff/templates/staff_user_import.html:46]: you don't need to include the data-label here and in the other new modals. they are needed for the deletion confirmation of individual entries to make sure the user deletes the correct item, but here we don't have to fear confusion because there are not many entries that all look alike.

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 6]
```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
...
+            var input = $("<input>")
+            .attr("type", "hidden")
+            .attr("name", "operation").val(dataId);
+            $('#contributor-import-form').append(input);
+            $('#contributor-import-form').submit(); 
+        };
+    </script>
+
+    {% trans 'Copy contributors' as title %}
+    {% blocktrans asvar question %}Do you really want to copy contributors? {% endblocktrans %}
```
Comment (janno42) [inline evap/staff/templates/staff_evaluation_person_management.html:168]: please make it `Do you really want to copy the contributors?` (with `the` and without a trailing space).

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 7]
```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
...
+            var input = $("<input>")
+               .attr("type", "hidden")
+               .attr("name", "operation").val(dataId);
+            $('#participant-import-form').append(input);
+            $('#participant-import-form').submit();
+        };
+    </script>
+
+    {% trans 'Copy participants' as title %}
+    {% blocktrans asvar question %}Do you really want to copy participants? {% endblocktrans %}
```
Comment (janno42) [inline evap/staff/templates/staff_evaluation_person_management.html:140]: please make it `Do you really want to copy the participants?` (with `the` and without a trailing space).

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 8]
```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
...
+            var input = $("<input>")
+            .attr("type", "hidden")
+            .attr("name", "operation").val(dataId);
+            $('#participant-copy-form').append(input);
+            $('#participant-copy-form').submit();  
+        };
+    </script>
+
+    {% trans 'Import contributors' as title %}
+    {% blocktrans asvar question %}Do you really want to import the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
```
Comment (He3lixxx) [inline evap/staff/templates/staff_evaluation_person_management.html:154]: trailing space here too

File: evap/staff/templates/staff_evaluation_person_management.html [hunk 9]
```diff
@@ -120,6 +120,66 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
     </div>
 {% endblock %}
 
+{% block modals %}
+{{ block.super }}
+    {% trans 'Import participants' as title %}
+    {% blocktrans asvar question %}Do you really want to import the Excel file with <strong id="data-label"></strong>? {% endblocktrans %}
```
Comment (He3lixxx) [inline evap/staff/templates/staff_evaluation_person_management.html:126]: trailing space inside the translation

File: evap/staff/templates/staff_semester_import.html [hunk 10]
```diff
@@ -34,13 +34,31 @@ <h3>{% trans 'Import semester data' %}</h3>
                     <button name="operation" value="test" type="submit" class="btn btn-primary form-submit-btn">{% trans 'Upload and test' %}</button>
```
Comment (janno42) [inline evap/staff/templates/staff_semester_import.html:34]: this button still uses the wrong class
